content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -1,52 +1,114 @@
+support at-rest encryption. This at-rest encryption is additional to any system-level
+encryption for the etcd cluster or for the filesystem(s) on hosts where you are running the
+kube-apiserver.
+
+This page shows how to enable and configure encryption of API data at rest.
+
+{{< note >}}
+This task covers encryption for data stored using the
+{{< glossary_tooltip text="Kubernetes API" term_id="kubernetes-api" >}}.
+
+If you want to encrypt data in filesystems that are mounted into containers, you instead need
+to either:
+
+- use a storage integration that provides encrypted
+{{< glossary_tooltip text="volumes" term_id="volume" >}}
+- encrypt the data within your own application
+{{< /note >}}
 
 ## {{% heading "prerequisites" %}}
 
-* {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+* {{< include "task-tutorial-prereqs.md" >}}
+
+* This task assumes that you are running the Kubernetes API server as a
+  {{< glossary_tooltip text="static pod" term_id="static-pod" >}} on each control
+  plane node.
 
-* etcd v3.0 or later is required
+* Your cluster's control plane **must** use etcd v3.x (major version 3, any minor version).
 
 * To encrypt a custom resource, your cluster must be running Kubernetes v1.26 or newer.
+  {{< version-check >}}
 
 
 <!-- steps -->
 
-## Configuration and determining whether encryption at rest is already enabled
+## Determine whether encryption at rest is already enabled {#determining-whether-encryption-at-rest-is-already-enabled}
+
+By default, the API server uses an `identity` provider to protect confidential data
+within etcd.
+**The default `identity` provider does not provide any confidentiality protection.**
 
 The `kube-apiserver` process accepts an argument `--encryption-provider-config`
-that controls how API data is encrypted in etcd.
-The configuration is provided as an API named
-[`EncryptionConfiguration`](/docs/reference/config-api/apiserver-encryption.v1/). `--encryption-provider-config-automatic-reload` boolean argument determines if the file set by `--encryption-provider-config` should be automatically reloaded if the disk contents change. This enables key rotation without API server restarts. An example configuration is provided below.
+that specifies a path to a configuration file. The contents of that file, if you specify one,
+control how Kubernetes API data is encrypted in etcd.
+
+The format of that configuration file is YAML, representing a configuration API kind named
+[`EncryptionConfiguration`](/docs/reference/config-api/apiserver-encryption.v1/).
+You can see an example configuration
+in [Encryption at rest configuration](#understanding-the-encryption-at-rest-configuration).
+
+Check whether the `kube-apiserver` process is running with the `--encryption-provider-config`
+command line argument set to `true`. If it is not, you do not have encryption at rest enabled.
+Otherwise, you need to perform further checks to confirm the situation.
```

Comment:
line 64 is confusing... what am I supposed to do?

Reply:
```suggestion
Otherwise, you need to perform further checks to confirm the situation. Read on to learn
about these further checks.
```
?

Reply:
I'd suggest we remove this paragraph and add one sentence after line 55:

If it is not specified, you do not have encryption at rest enabled.